picard/ui/searchdialog/album.py

```diff
@@ -96,6 +99,12 @@ def showEvent(self, event):
         super().showEvent(event)
         self.shown.emit()
 
+    def invalidate(self):
+        self.layout = None
+
+    def __del__(self):
+        self.shown.disconnect()
```

Comment:
Why is this needed? This should happen automatically. Or was there some issue still producible without this change?.

Reply:
It was a leftover from experiments, unneeded.